homeassistant/components/overkiz/__init__.py

```diff
@@ -144,3 +147,62 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.data[DOMAIN].pop(entry.entry_id)
 
     return unload_ok
+
+
+async def _async_migrate_entries(
+    hass: HomeAssistant, config_entry: ConfigEntry
+) -> bool:
+    """Migrate old entries to new unique IDs."""
+    entity_registry = er.async_get(hass)
+
+    @callback
+    def update_unique_id(entry: er.RegistryEntry) -> dict[str, str] | None:
+        # Python 3.11 treats (str, Enum) and StrEnum in a different way
+        # Since pyOverkiz switched to StrEnum, we need to rewrite the unique ids once to the new style
+        #
+        # io://xxxx-xxxx-xxxx/3541212-OverkizState.CORE_DISCRETE_RSSI_LEVEL -> io://xxxx-xxxx-xxxx/3541212-core:DiscreteRSSILevelState
```

Comment:
Please break long comments at maximum 88 characters per line. Here I'd break after the arrow.

Reply:
Happy to do. Should this not be part of our pre-commit / validation scripts if this is preferred? I use auto formatting and pre-commit for all of my commits, and enforcing certain rules would help.

Reply:
I think there is already many lines that are too long in our code so we can't validate all existing code like that until black can format it automatically.

Reply:
Makes sense. I will fix this in a follow up PR to make sure Overkiz is at least 'compliant'. Thanks.